test/cri-containerd/container_test.go

```diff
@@ -221,6 +221,133 @@ func Test_RunContainer_Events_LCOW(t *testing.T) {
 	}
 }
 
+func Test_RunContainer_VirtualDevice_LocationPath_WCOW_Process(t *testing.T) {
+	requireFeatures(t, featureWCOWProcess)
+
+	testDeviceLocationPath, err := findTestNvidiaGPULocationPath()
+	if err != nil {
+		t.Skipf("skipping test, failed to retrieve assignable device on host with: %v", err)
+	}
+	if testDeviceLocationPath == "" {
+		t.Skipf("skipping test, host has no assignable devices")
+	}
+
+	pullRequiredLcowImages(t, []string{imageLcowK8sPause, imageLcowAlpine})
+	client := newTestRuntimeClient(t)
+
+	podctx := context.Background()
+	sandboxRequest := &runtime.RunPodSandboxRequest{
+		Config: &runtime.PodSandboxConfig{
+			Metadata: &runtime.PodSandboxMetadata{
+				Name:      t.Name(),
+				Namespace: testNamespace,
+			},
+		},
+		RuntimeHandler: wcowProcessRuntimeHandler,
+	}
+
+	podID := runPodSandbox(t, client, podctx, sandboxRequest)
+	defer removePodSandbox(t, client, podctx, podID)
+	defer stopPodSandbox(t, client, podctx, podID)
+
+	device := &runtime.Device{
+		HostPath: "vpci-location-path://" + testDeviceLocationPath,
+	}
+
+	containerRequest := &runtime.CreateContainerRequest{
+		Config: &runtime.ContainerConfig{
+			Metadata: &runtime.ContainerMetadata{
+				Name: t.Name() + "-Container",
+			},
+			Image: &runtime.ImageSpec{
+				Image: imageWindowsNanoserver,
+			},
+			Command: []string{
+				"cmd",
+				"/c",
+				"ping",
+				"-t",
+				"127.0.0.1",
+			},
+			Devices: []*runtime.Device{
+				device,
+			},
+		},
+		PodSandboxId:  podID,
+		SandboxConfig: sandboxRequest.Config,
+	}
+	ctx, cancel := context.WithTimeout(context.Background(), 20*time.Second)
+	defer cancel()
+
+	containerID := createContainer(t, client, ctx, containerRequest)
+	runContainerLifetime(t, client, ctx, containerID)
+}
+
+func Test_RunContainer_VirtualDevice_ClassGUID_WCOW_Process(t *testing.T) {
+	requireFeatures(t, featureWCOWProcess)
```

Comment:
Should these be updated with `featureGPU`?

Reply:
I'll merge https://github.com/microsoft/hcsshim/pull/805 then rebase this and update!